edenia/antelope-tools
# Chore/ block history

### Save the block history for the last 90 days

### What does this PR do?

- Clear blocks older than 90 days from libre testnet block history
- Resolve #847

### Steps to test

1. Run the project locally
2. Check the table block_history
3. Go to the block distribution page
4. Check the graphic

### Screenshot

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/5c278840-27e1-4b3d-98f3-188e57facf2b)
